diff --git a/node_modules/react-multi-select-component/src/lib/interfaces.ts b/node_modules/react-multi-select-component/src/lib/interfaces.ts
index c7d8a37..9d402ad 100644
--- a/node_modules/react-multi-select-component/src/lib/interfaces.ts
+++ b/node_modules/react-multi-select-component/src/lib/interfaces.ts
@@ -11,6 +11,7 @@ export interface ITheme {
 export interface Option {
   value;
   label: string;
+  name : string;
   key?: string;
   disabled?: boolean;
 }
diff --git a/node_modules/react-multi-select-component/src/select-panel/default-item.tsx b/node_modules/react-multi-select-component/src/select-panel/default-item.tsx
index 7f36be9..7ce5ab3 100644
--- a/node_modules/react-multi-select-component/src/select-panel/default-item.tsx
+++ b/node_modules/react-multi-select-component/src/select-panel/default-item.tsx
@@ -41,7 +41,7 @@ const DefaultItemRenderer = ({
         tabIndex={-1}
         disabled={disabled}
       />
-      <span>{option.label}</span>
+      <span>{option.name}</span>
     </div>
   );
 };
